pdc/apps/auth/views.py

```diff
@@ -134,6 +140,142 @@ def user_profile(request):
     return render(request, 'user_profile.html', context)
 
 
+def get_users_and_groups(arg):
+    # get all groups
+    members = dict()
+    groups_list = []
+    try:
+        group_resource_permission_list = get_list_or_404(models.GroupResourcePermission,
+                                                         resource_permission=arg)
+        for group_resource_permission in group_resource_permission_list:
+            groups_list.append(str(group_resource_permission.group.name))
+    except Http404:
+        pass
+
+    # get all users
+    users_list = []
+    users = models.User.objects.all()
+    for user in users:
+        if user.is_superuser:
+            users_list.append(str(user.username))
+        else:
+            resource_permission_set = get_resource_permission_set(user)
```

Comment:
`get_resource_permission_set` also contain `is_superuser` judgement. Seems all could use logic here.

Besides, it looks pretty low efficiency. Even these users is gotten from groups, still need a loop for all users, maybe thousands times? Not sure. And just for one resource, and `get_resource_permission_set` also need to access DB several times. It looks very inefficient. I doubt if it could work well in real environment. 

Should we consider a non-super user get permissions out of his group now?